eox_nelp/pearson_vue/rti_backend.py

```diff
@@ -14,7 +17,10 @@
     handle_course_completion_status,
     import_candidate_demographics,
     import_exam_authorization,
+    validate_cdd_request,
+    validate_ead_request,
```

Comment:
you are validating after sending the request

Reply:
These are the imports I think so

Reply:
hahah you are right